packages/agoric-cli/README.md

```diff
@@ -42,14 +36,14 @@ Here is a simple script for how to get started.
 ```sh
 # Initialize your dapp project.
 # Note: Change the `demo` name to something meaningful.
-agoric init demo
+$AGORIC init demo
 # Go to its directory.
 cd demo
-# Install Javascript/Go dependencies.
-agoric install
+# Install Javascript dependencies.
+$AGORIC install
```

Comment:
I get the following, presumably because cosmic-swingset@^0.11.0 hasn't been released yet. Was this meant to succeed or are we planning on this failing for now? Or is there something else I should be doing?

```
warning Lockfile has incorrect entry for "@agoric/cosmic-swingset@^0.11.0". Ignoring it.
Couldn't find any versions for "@agoric/cosmic-swingset" that matches "^0.11.0"
? Please choose a version of "@agoric/cosmic-swingset" from this list: (Use arrow keys)
❯ 0.10.15 
  0.10.14 
  0.10.13 
  0.10.12 
  0.10.11 
  0.10.10 
  0.10.9 
  0.10.8 
  0.10.8-dev.0 
```

Reply:
I expect you to use the `packages/agoric-cli/template` folder directly, and just skip doing the `$AGORIC init`.  Until that works, there is no hope of `$AGORIC init` working.

Reply:
I've documented this as such.